src/components/Error/Error.tsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, FC } from 'react';
+
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Don't disable eslint. Try to fix the error

Reply:
i can't because need change eslint, but i don't have opportunity